### DISCUSSION TITLE
Welcome to kgtk Discussions!

### DISCUSSION THREAD

**Discussion by saggu** [usc-isi-i2/kgtk#369] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by szeke** [comment]
Hi, we welcome all users to ask questions here, we will do our best to answer them promptly.
